src/test/java/Homework18.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+    @Test()
+    public void PlaySong () throws InterruptedException {
+        navigateToPage();
+        provideEmail("pchelka240386@gmail.com");
+        providePassword ("te$t$tudent");
+        clickSubmit();
+        Thread.sleep(3000);
+        clickPlayNextBtn();
+        checkSongIsPlaying();
+    }
+
+    private void clickSubmit() {
+    }
+
+    private void providePassword(String te$t$tudent) {
+    }
+
+    private void provideEmail(String "pchelka240386@gmail.com"String mail) {
+    }
+
+    private void navigateToPage() {
+    }
+
+    private void checkSongIsPlaying() {
+        WebElement soundBar= driver.findElement(By.cssSelector("[data-testid='sound-bar-play']"));
```

Comment:
2. I don't see you define step initialize driver in this Homework18 script or `BaseTest `class, so can't perform any action on any element if driver is not initialized and lunch browser.
//HINT: You can define step launchBrowser() into BaseTest after all classes inheritance from `BaseTest `driver will be initialized

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/44592718/2564f3f7-cbe4-4398-bff5-724c2d24ed23)